src/pages/index.tsx

```diff
@@ -58,15 +72,35 @@ const HomePage: NextPageWithLayout = () => {
 
   if (signedIn || signedInOptimistic) {
     return (
-      <ComponentWrapperPage
-        src={components.tosCheck}
-        componentProps={{
-          logOut: authStore.logOut,
-          targetProps: componentProps,
-          targetComponent: components.default,
-          tosName: components.tosContent,
-        }}
-      />
+      <>
+        <VmComponent
+          src="near/widget/NearOrg.Notifications.NotificationAlert"
```

Comment:
After #408 done we need to update this string:
```suggestion
          src={components.nearOrg.notifications.notificationAlert}
```